### PR TITLE
Add QuadroSync v1.0.0

### PR DESCRIPTION
**Pull request by danielbayley**
QuadroSync to make your computer compatible with Quadro.

Signed-off-by: Daniel Bayley daniel.bayley@me.com

### CODE REVIEW

File: Casks/quadrosync.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  sha256 :no_check
+
+  url 'https://s3-eu-west-1.amazonaws.com/quadro-downloads/daemon/mac/QuadroSync_mac.dmg'
+  name 'QuadroSync'
+  homepage 'http://quadro.me/sync'
+  license :gratis
+
+  app "QuadroSync.app"
+
+  zap :delete => '/Library/*/com.quadro.*',
```
Comment (jawshooah) [inline Casks/quadrosync.rb]: We don't support glob expansion at this time. You'll have to provide the full path.

File: Casks/quadrosync.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cask 'quadrosync' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://s3-eu-west-1.amazonaws.com/quadro-downloads/daemon/mac/QuadroSync_mac.dmg'
```
Comment (jawshooah) [inline Casks/quadrosync.rb]: This requires a comment `# amazonaws.com is the official download host per the vendor homepage` ([example](https://github.com/caskroom/homebrew-cask/blob/77b529d0011fcc687852b04a5c4bb1c1edf10433/Casks/send-to-kindle.rb#L5)).

**Comment by jawshooah** [comment]
The style errors can be fixed with `bundle install && bundle exec rubocop --auto-correct`.

File: Casks/quadrosync.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  name 'QuadroSync'
+  homepage 'http://quadro.me/sync'
+  license :gratis
+
+  app "QuadroSync.app"
```
Comment (vitorgalvao) [inline Casks/quadrosync.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

Comment (vitorgalvao) [inline Casks/quadrosync.rb]: Downloading it, this is wrong. It needs to use [`installer :manual`](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#installer-manual) and requires [`uninstall`](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details).

`installer :manual => 'QuadroInstaller.app'`.

File: Casks/quadrosync.rb (hunk 1 above)
Comment (danielbayley) [inline Casks/quadrosync.rb]: This was meant to remove 2 files along the same path which share a very similar name, but actually `uninstall` `:launchctl` applies to one of them anyway.

There are some existing formulas that use[d] glob expansion… Will they be broken now?

**Comment by jawshooah** [comment]
@danielbayley where are you seeing globs used elsewhere?

**Comment by danielbayley** [comment]
@jawshooah
```
pwd
/usr/local/Library/Taps/caskroom/homebrew-cask/Casks

grep -R '\/\*' .
./ableton-live-standard.rb:  zap :delete => '~/Library/*/*[Aa]bleton*',
./ableton-live-suite.rb:  zap :delete => '~/Library/*/*[Aa]bleton*',
./ableton-live.rb:  zap :delete => '~/Library/*/*[Aa]bleton*',
./chefdk.rb:  #  find /usr/local/bin -lname '/opt/chefdk/*'
./gedit.rb:    library = Dir.glob("#{`brew --cellar`.chomp}/libxml2/**/libxml2.2.dylib").first
./kicad.rb:    system '/bin/mv', '--', "#{staged_path}/Kicad/*.app", "#{staged_path}/Kicad-apps/"
./livesurface-context.rb:  zap :delete => '~/Library/*/*[Ll]ive[Ss]urface*'
./pl2303.rb:                          '/var/db/receipts/*PL2303*.*',
./pl2303.rb:                          '/var/db/receipts/*ProlificUSbSerial*.*',
./programmer-dvorak.rb:                          # todo expand/glob for '/private/var/folders/*/*/-Caches-/com.apple.IntlDataCache.le*',
./splice.rb:                   '~/Library/Application Support/*Splice*',
./touchosc-editor.rb:  zap :delete => '~/Library/*/*TouchOSCEditor*'
```

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in d85d090bc92478c0151be2f464badb34a39fcfc3. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**